.npmrc

```diff
@@ -1 +0,0 @@
-tag-version-prefix=""
```

Comment:
The tag version prefix was necessary for doing the public release to npm registry. Is it obsolete?

Reply:
I don't actually know - it looked like Dudley introduced this with his last PR and he said it was irrelevant, so I removed it. If we still need it, I can put it back in

Reply:
![image](https://user-images.githubusercontent.com/39338561/85860539-40155d80-b7bf-11ea-9472-97e15f830117.png)